### PR TITLE
Get Account API: Get username/email from ID

### PR DESCRIPTION
**Pull request by shashankmehra**
Update documentation

### CODE REVIEW

File: api/accounts/get_account_test.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	})
+
+	t.Run("valid account", func(t *testing.T) {
+		account, err := app.AccountStore.Create("unlocked@test.com", []byte("bar"))
+		require.NoError(t, err)
+
+		res, err := client.Get(fmt.Sprintf("/accounts/%v", account.ID))
+		require.NoError(t, err)
+		assert.Equal(t, http.StatusOK, res.StatusCode)
+		test.AssertGetAccountResponse(t, res, account)
```
Comment (cainlevy) [inline api/accounts/get_account_test.go]: i don't anticipate this being reused. would you please make it a private func in this file?

File: docs/guide-implementing_signup.md [hunk 2]
```diff
@@ -32,6 +32,11 @@ process that submits with two API calls.
 > the data client-side (step 3) is important because it will significantly reduce the odds of this
 > happening.
 
+> NOTE:
```
Comment (cainlevy) [inline docs/guide-implementing_signup.md]: 💯 for adding these docs. thanks!

**Comment by shashankmehra** [comment]
@cainlevy Made the changes.

**Comment by shashankmehra** [comment]
Weird. Its passing locally but it should not. Fixing.

**Comment by shashankmehra** [comment]
@cainlevy The function `AssertGetAccountResponse` uses `extractResult` which is part of `asserts.go`. To move `AssertGetAccountResponse` would require to copy that function there too. I think to avoid code duplication we should let it be. Even though that function will not be reused, it would make sense to have it there from code organisation point of view. Let me know if you still want to move it.

**Comment by cainlevy** [comment]
thanks for the pr! 🎉

**Pull request merged**